replit_river/rpc.py

```diff
@@ -44,27 +44,30 @@
 STREAM_OPEN_BIT = 0x0002
 STREAM_CLOSED_BIT = 0x0004
 
-# Equivalent of https://github.com/replit/river/blob/c1345f1ff6a17a841d4319fad5c153b5bda43827/transport/message.ts#L23-L33
+# retriable codes
```

Comment:
we need to define what "retriable" means here. e.g. HTTP makes a distinction between "you can retry the exact same request a-la 5XX" and "you can retry but only if you modify your request a-la 400, so it's not _really_ retriable".

by that distinction, this is more 4XX, and therefore are non-retriable in spirit.

Reply:
hm yeah seems like theres 3

1. retry safely without doing anything (nothing seems to really fall under here)
2. retry with a new session (session mismatch)
3. pls don't retry (protocol mismatch, etc.)

we only seem to have 2 and 3 cases for handshake so calling it retriable might be fine?

Reply:
we shouldn't give it an ambiguous descriptor: we don't have a constraint on number of words in a comment, so might as well spell out explicitly what this means from a semantic perspective!